### ISSUE TITLE
how does the downwardedges example work?

### ISSUE THREAD

**Issue by aspiers** [tgdwyer/WebCola#60]
I previously thought I understood http://marvl.infotech.monash.edu/webcola/examples/downwardedges.html but now I'm totally confused.  The `chris.json` data includes separation constraints, but they are not fed into cola via `d3cola.constraints(...)` ?!  So aren't they ignored?  Is something else responsible for setting up the constraints, or what?

**Comment by aspiers** [comment]
I guess `.flowLayout("y", 30)` sets up the separation constraints.  According to http://marvl.infotech.monash.edu/webcola/examples/unix.html:

> In particular, the call to `flowLayout` causes all edges not involved in a cycle (strongly connected component) to have a separation constraint generated between their source and sink, with a minimum spacing set to 150. Specifying the '`x`' axis achieves a left-to-right flow layout. The default is top-to-bottom flow layout (i.e. '`y`'). You can also specify a function as the second argument to `flowLayout` that returns different separations for each edge. Specify a different minimum as a second parameter to `flowLayout`.

But this still doesn't explain why `chris.json` has a load of constraints which are ignored.

**Comment by tgdwyer** [comment]
Sorry for the confusion.  I originally specified the constraints in chris.json manually.  When I implemented the `flowLayout` stuff, which generates separation constraints automatically over edges not involved in strongly-connected components (i.e. cycles), I switched the downwardedges example over to use it without removing the separation constraints from `chris.json`.  

I'm going to leave the constraints in chris.json as an example of how to specify separation constraints manually and I've added a comment to downward edges to explain the two ways to achieve such constraints (either manually or through `flowLayout`).
